crates/turborepo-lib/src/cli/error.rs

```diff
@@ -53,3 +55,51 @@ pub enum Error {
     #[diagnostic(transparent)]
     Watch(#[from] watch::Error),
 }
+
+const MAX_CHARS_PER_TASK_LINE: usize = 100;
+
+pub async fn print_potential_tasks(
+    base: CommandBase,
+    telemetry: CommandEventBuilder,
+) -> Result<(), Error> {
+    let signal = get_signal()?;
+    let handler = SignalHandler::new(signal);
+    let ui = base.ui;
+
+    let run_builder = RunBuilder::new(base)?;
+    let run = run_builder.build(&handler, telemetry).await?;
+    let potential_tasks = run.get_potential_tasks()?;
+
+    println!("No tasks provided, here are some potential ones to run\n",);
+
+    for (task, packages) in potential_tasks
+        .into_iter()
+        .sorted_by(|(_, a), (_, b)| b.len().cmp(&a.len()))
```

Comment:
question: totally for my own learning, is there any benefit to doing this over

```rs
.sorted_by_key(|&(_, ref value)| value.len()).rev()
```

Reply:
If I had to guess, the `rev` call will collect the iterator into a vector then turn it back into an iterator (since there's no other way to reverse an iterator), so this will cause a couple more allocations. Not a huge deal, and possibly optimized by the compiler, but this is a little more direct.

Reply:
thanks!